wire-runtime/src/commonMain/kotlin/com/squareup/wire/MessageSerializedForm.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire
+
+import com.squareup.wire.internal.ObjectStreamException
+import com.squareup.wire.internal.Serializable
+import com.squareup.wire.internal.Throws
+import kotlin.reflect.KClass
+
+internal expect class MessageSerializedForm<M : Message<M, B>, B : Message.Builder<M, B>>(
```

Comment:
This really shouldn't be in commonMain. I realize it requires expect/actual-ing Message, but I'm not sure dragging in this an serializable and object stream exception is any better.

Reply:
Agree, looks like I won't be able to avoid expect/actual without hacks. I'll experiment with slimming down `Builder` since generated Kotlin code doesn't need it, this might help make JS and Native actuals pretty small.